packages/naming.entity.stringify/index.js

```diff
@@ -42,8 +42,9 @@ function stringify(entity, delims) {
  * @returns {Function}
  */
 function stringifyWrapper(convention) {
-    console.assert(convention.delims && convention.delims.elem && convention.delims.mod,
-        '@bem/sdk.naming.entity.stringify: convention should be an instance of BemNamingEntityConvention');
+    // TODO(yarastqt): Use assert library
```

Comment:
No need to use assert library when language already has some built-in version.
Need to create an issue with detailed description why we can't use it here and put link in TODO